build/testdependencies.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+    <Import Project="dependencies.props" />
```

Comment:
Any reason why we can't just define these in the same dependencies.props file?  We could label the PropertyGroup if that would make it more clear?